mypyc/primitives/registry.py

```diff
@@ -368,12 +370,15 @@ def c_method_op(name: str,
         truncated_type: type to truncated to(See Truncate for info)
                         if it's defined both return_type and it should be non-referenced
                         integer types or bool type
+        ordering: optional ordering of the arguments, if defined,
+                  reorders the arguments accordingly.
+                  should never be used together with var_arg_type
```

Comment:
Also mention that all the other arguments, such as `arg_types`, are in the order accepted by the called function (i.e., after reordering).

Reply:
I am a bit confused about this. `arg_types` are actually accepted by the python scope function(opposed to the called C function) and `ordering` is actually reordering `arg_types`

Reply:
Sorry, I was confused. The order of all the other arguments are *before* reordering (Python ordering), right? Anyway, this illustrates that this is useful to document!